app/api/routers/breeze_buddy/configurations/handlers.py

```
@@ -0,0 +1,286 @@
+    logger.info(
+        f"Admin {current_user.username} updating configuration: {config_id}"
+    )
+
+    try:
+        # Verify configuration exists
+        existing_config = await get_call_execution_config_by_id(config_id)
+        if not existing_config:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail=f"Configuration {config_id} not found"
+            )
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)